PowerEditor/src/NppDarkMode.cpp

```diff
@@ -983,7 +983,7 @@ namespace NppDarkMode
 		GetThemeBackgroundContentRect(buttonData.hTheme, hdc, BP_GROUPBOX, iStateID, &rcBackground, &rcContent);
 		ExcludeClipRect(hdc, rcContent.left, rcContent.top, rcContent.right, rcContent.bottom);
 
-		DrawThemeParentBackground(hwnd, hdc, &rcClient);
+		//DrawThemeParentBackground(hwnd, hdc, &rcClient);
```

Comment:
This is causing visual glitch, currently only commented, in case there is some side effect.
But I have not noticed anything for now.